### ISSUE TITLE
Use  arrow keys to move in drop-down list

### ISSUE THREAD

**Issue by Chesla** [arendjr/selectivity#163]
I am not able to use arrow keys to move in drop-down list. Can any such functionality be present?

**Comment by arendjr** [comment]
Yes, it should be. I'm assuming you're not using a custom build? If you are you should include the keyboard module to get this functionality. I'll check to see if anything else might be broken...

**Comment by Chesla** [comment]
Ya, please look at the matter and confirm me
